Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -8,12 +8,16 @@
 using MbUnit.Framework;
 using Telerik.Sitefinity.Abstractions;
 using Telerik.Sitefinity.Frontend.Navigation.Mvc.Controllers;
+using Telerik.Sitefinity.Frontend.Navigation.Mvc.Models.LanguageSelector;
+using Telerik.Sitefinity.Frontend.News.Mvc.Controllers;
 using Telerik.Sitefinity.Frontend.TestUtilities;
 using Telerik.Sitefinity.Localization;
 using Telerik.Sitefinity.Modules.Pages;
 using Telerik.Sitefinity.Mvc.Proxy;
+using Telerik.Sitefinity.News.Model;
 using Telerik.Sitefinity.Pages.Model;
 using Telerik.Sitefinity.TestIntegration.Data.Content;
+using Telerik.Sitefinity.TestUtilities.CommonOperations;
```

Comment:
When using {namespace} is added there is no need to use the namespace when calling classes from this namespace. ServerOperations call be used without the namespace in front of the class name.